infrastructure/conf/nginx-conf-local/default

```diff
@@ -223,6 +223,16 @@ server {
         rewrite "^(?:/[a-z]{2})?/support(/.*|$)" /apps/web/dist/index.html break;
     }
 
+    # rewrite possible lang prefix then "support" to apps/web index.html
```

Comment:
```suggestion
    # rewrite possible lang prefix then "terms-and-conditions" to apps/web index.html
```

Reply:
Nice catch! Fixed by [72efc27](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9143/commits/72efc27d358293dd6ee929d32511c81a85b0fbe2)

This applies to the requested changes below as well!